tests/components/datadog/test_init.py

```diff
@@ -10,59 +9,47 @@
     STATE_ON,
 )
 import homeassistant.core as ha
-from homeassistant.setup import setup_component
+from homeassistant.setup import async_setup_component
 
-from tests.common import assert_setup_component, get_test_home_assistant
+from tests.async_mock import MagicMock, patch
+from tests.common import assert_setup_component
 
 
-class TestDatadog(unittest.TestCase):
-    """Test the Datadog component."""
+async def test_invalid_config(hass):
+    """Test invalid configuration."""
+    with assert_setup_component(0):
+        assert not await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)